tests/UnifiedSpecTests/Operation.php

```diff
@@ -211,6 +211,7 @@ private function execute()
     private function executeForChangeStream(ChangeStream $changeStream)
     {
         $args = $this->prepareArguments();
+        Util::assertParamsBySchema(get_class($changeStream), $this->name, $args);
```

Comment:
Let's use the `::class` constant instead of `get_class` in all of these contexts.

```suggestion
        Util::assertParamsBySchema(ChangeStream::class, $this->name, $args);
```

I'll leave you to revise the other occurrences.